git/git.go

```diff
@@ -307,6 +307,19 @@ func CheckoutBranch(branch string) error {
 	return run.PrepareCmd(configCmd).Run()
 }
 
+// pull changes from remote branch without version history
+func RunPull(baseRemote, baseBranch string) error {
```

Comment:
Nit: the remote and branch arguments could just be called `remote` and `branch`. The "base" prefix has no meaning in the context of this lower-level function.